src/test/java/bdd/steps/StepDefinitions.java

```diff
@@ -0,0 +1,38 @@
+package bdd.steps;
+
+import driver.DriverManager;
+import io.cucumber.java.After;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+import pages.cucumber.CucumberToolsPage;
+import pages.cucumber.CucumberHomePage;
+
+public class StepDefinitions {
+
+    @After
+    public void closeBrowser() {
+        DriverManager.closeDriver();
+    }
+
+    @When("the user opens Cucumber website")
```

Comment:
@Given, not @When

Reply:
Changed, thank you for pointing.
Please see in the following commit.